pkg/scheduler/factory_test.go

```diff
@@ -15,308 +15,3 @@ limitations under the License.
 */
 
 package scheduler
-
-import (
-	"context"
-	"errors"
-	"fmt"
-	"testing"
-	"time"
-
-	"github.com/google/go-cmp/cmp"
-	v1 "k8s.io/api/core/v1"
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/sets"
-	"k8s.io/client-go/informers"
-	"k8s.io/client-go/kubernetes/fake"
-	"k8s.io/client-go/tools/cache"
-	extenderv1 "k8s.io/kube-scheduler/extender/v1"
-	"k8s.io/kubernetes/pkg/scheduler/framework"
-	internalcache "k8s.io/kubernetes/pkg/scheduler/internal/cache"
-	internalqueue "k8s.io/kubernetes/pkg/scheduler/internal/queue"
-	testingclock "k8s.io/utils/clock/testing"
-)
-
-const (
-	podInitialBackoffDurationSeconds = 1
-	podMaxBackoffDurationSeconds     = 10
```

Comment:
Removed `podInitialBackoffDurationSeconds` and `podMaxBackoffDurationSeconds` as we don't use them in any place.